rest/replicatortest/replicator_collection_test.go

```diff
@@ -277,6 +276,8 @@ func TestActiveReplicatorMultiCollection(t *testing.T) {
 		assert.Len(t, changes.Results, expectedNumDocs)
 	}
 
+	require.NoError(t, ar.Stop())
```

Comment:
is it worth adding a defer on stop without checking error to make sure it is stopped?